app/models/invitation.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Invitation < ApplicationRecord
+  EXPIRE_TIME_LIMIT = 15.minutes
```

Comment:
We don`t need EXPIRE_TIME, if you have invitation link it should work all time

Reply:
removed

Reply:
removed expire time